keyboards/svalboard/keymaps/keymap_support.c

```diff
@@ -201,19 +200,20 @@ bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
         in_mod_tap_layer = get_highest_layer(layer_state);
         layer_state = layer_state & ~(1 << in_mod_tap_layer);
 
-        action_exec(record->event);
-
+//        action_exec(record->event);
```

Comment:
I'd prefer to not leave code commented out with no explanation, because it makes my brain nervous wondering if it's meant to be that way.  :laughing:  I say purge it or explain it.